### ISSUE TITLE
restore dagster-aws

### ISSUE THREAD

**Issue by bollwyvl** [conda-forge/dagster-feedstock#78]
As of #77, as it appears the run/test environments for ~~`dagster-snowflake` and~~ `dagster-aws` have become unsolveable. Some more careful study of the upstreams is likely required.

**Comment by bollwyvl** [comment]
#100 brings back :snowflake:, aws was a wash...

**Comment by bollwyvl** [comment]
It looks like some of the complexity may have been reduced around aws in `0.13.4`...

```diff
--- a/python_modules/libraries/dagster-aws/setup.py
+++ b/python_modules/libraries/dagster-aws/setup.py
@@ -36,10 +36,10 @@ def get_version() -> str:
             "boto3",
             f"dagster{pin}",
             "packaging",
-            "psycopg2-binary",
             "requests",
         ],
         extras_require={
+            "redshift": ["psycopg2-binary"],
             "pyspark": ["dagster-pyspark"],
             "test": ["moto==1.3.16", "requests-mock"],
         },
diff --git a/python_modules/libraries/dagster-dask/setup.py b/python_modules/libraries/dagster-dask/setup.py
index 75a1fd1..ce514ab 100644
--- a/python_modules/libraries/dagster-dask/setup.py
+++ b/python_modules/libraries/dagster-dask/setup.py
@@ -36,9 +36,6 @@ def get_version() -> str:
             f"dagster{pin}",
             "dask[dataframe]>=1.2.2",
             "distributed>=1.28.1",
-            # resolve issue with aiohttp pin of chardet for aiohttp<=3.7.3, req'd by dask-kubernetes
-            # https://github.com/dagster-io/dagster/issues/3539
-            "chardet<4.0",
         ],
         extras_require={
             "yarn": ["dask-yarn"],

```

**Comment by bollwyvl** [comment]
closed by #194
